src/Sulu/Component/Content/Mapper/ContentMapperInterface.php

```diff
@@ -82,11 +82,10 @@ public function saveExtension(
      * @param string $languageCode Save data for given language
      * @param int $userId The id of the user who saves
      * @param bool $partialUpdate ignore missing property
+     * @param bool $isShadow
+     * @param string $shadowBaseLanguage
      *
-     * @throws \PHPCR\ItemExistsException if new title already exists
-     * @throws \Exception
```

Comment:
only the exception is not thrown but the phpcr and the \InvalidArgumentException are thrown i will update it!